src/Pages/Sign/InputWithLabel.scss

```
@@ -0,0 +1,34 @@
+.idBox {
+  display: flex;
+  width: 100%;
+  padding: 7px 0px;
+  .formLabel {
+    width: 160px;
+    height: 46px;
+    color: #a36d6a;
+    font-weight: 700;
+    line-height: 46px;
+    .requiredMark {
+      margin-left: 5px;
+      color: #a36d6a;
+      font-weight: 500;
+      line-height: 46px;
+    }
+  }
+  .inputBox {
+    display: flex;
+    width: 700px;
+    input {
+      border: 1px solid #c8a7a6;
+      width: 300px;
+      height: 46px;
+    }
+    .description {
+      font-size: 13px;
+      color: #999;
+      height: 30px;
+      margin-top: 15px;
+      margin-left: 15px;
```

```suggestion
    .description {
      height: 30px;
      margin-top: 15px;
      margin-left: 15px;
      font-size: 13px;
      color: #999;
```
레이아웃에 영향을 크게 미치는 속성부터 위에 적어주시고 아래로 갈 수록 덜 중요한 정보를 배치해주시는게 좋겠습니다.